src/trading/utils.ts

```diff
@@ -76,3 +76,17 @@ export function mapQuoteAmountsAndCosts<T, R>(
     afterSlippage: serializeAmounts(value.afterSlippage),
   }
 }
+
+/**
+ * Set sell token to the initial one
+ * Because for ETH-flow orders we do quote requests with wrapped token
+ */
+export function getTradeParametersAfterQuote({
+  quoteParameters,
+  orderParameters,
+}: {
+  quoteParameters: TradeParameters
+  orderParameters: TradeParameters
+}): TradeParameters {
+  return { ...quoteParameters, sellToken: orderParameters.sellToken }
```

Comment:
Does this also take care of the `feeZero` issue?
Meaning, the quote returns a fee amount, but the order should not set a fee amount.

Reply:
Great, question.
Yes, it is hardcoded here: https://github.com/cowprotocol/cow-sdk/blob/main/src/trading/getOrderToSign.ts#L64